### PR TITLE
Update Qidi profiles, fixes thumbnails for X3 machines

### PR DESCRIPTION
**Pull request by HYzd766**
change of thumbnails

### CODE REVIEW

File: resources/profiles/Qidi/machine/fdm_qidi_x3_common.json [hunk 1]
```diff
@@ -19,11 +19,9 @@
       "0.4"
   ],
   "thumbnails": [
-    "380x380/COLPIC",
-    "210x210/COLPIC",
-    "380x380/PNG"
+    "380x380/ColPic",
```
Comment (SoftFever) [inline resources/profiles/Qidi/machine/fdm_qidi_x3_common.json:22]: 现在已经改成全大写了

File: resources/profiles/Qidi/machine/fdm_qidi_x3_common.json [hunk 2]
```diff
@@ -19,11 +19,9 @@
       "0.4"
   ],
   "thumbnails": [
-    "380x380/COLPIC",
-    "210x210/COLPIC",
-    "380x380/PNG"
```
Comment (SoftFever) [inline resources/profiles/Qidi/machine/fdm_qidi_x3_common.json:24]: PNG还需要的吧，否者fluidd页面里会缺失预览图

**Comment by HYzd766** [comment]
已经修改

**Review by SoftFever — APPROVED** [review]
LGTM
Thank you

**Comment by dewi-ny-je** [comment]
@HYzd766 why the change from crosshatch to grid? was it intended or did it get in by mistake?

**Pull request merged**